src/shared/format-table.c

```diff
@@ -2155,6 +2155,9 @@ int table_print(Table *t, FILE *f) {
                                                 if (!aligned)
                                                         return -ENOMEM;
 
+                                                if (j == display_columns - 1 && d->align_percent == 0)
+                                                        delete_trailing_chars(aligned, NULL);
```

Comment:
hmm, sounds pointless to first add the space, and then drop it again.  The space should not be added at all in the first place. 

moreover i guess the whitespace should also be suppressed for any alignment